test/functional/tools/mulled_example_multi_incomplete.xml

```diff
@@ -0,0 +1,19 @@
+<tool id="mulled_example_multi_incomplete" name="mulled_example_multi_incomplete" version="0.1.0">
+    <command><![CDATA[
+        bedtools --version > $out_file1 ;
+        echo "Moo" >> $out_file1 ;
+        samtools >> $out_file1 2>&1 ;
+        echo "Cow" >> $out_file1 ;
+    ]]></command>
+    <requirements>
+        <requirement type="package" version="1.3.1">samtools</requirement>
+        <requirement type="package" version="2.26.0">bedtools</requirement>
+        <requirement type="package" version="4.5">notarealpackage</requirement>
```

Comment:
Without this the Conda dependencies are all evaluated together using "resolve_all" and so an environment is never created I don't think - at least not a per-job environment. I can update this to add a comment about this.